contracts/distribution/subdomains/pricing/ZNSFixedPricing.sol

```diff
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.18;
+
+import { AZNSPricing } from "../abstractions/AZNSPricing.sol";
+import { IZNSRegistry } from "../../../registry/IZNSRegistry.sol";
+import { AAccessControlled } from "../../../access/AAccessControlled.sol";
+import { ARegistryWired } from "../../../abstractions/ARegistryWired.sol";
+
+
+contract ZNSFixedPricing is AAccessControlled, ARegistryWired, AZNSPricing {
```

Comment:
Not sure what you exactly mean here. Consolidate, meaning, put all options in one contract? That would defeat the purpose of "modular" system, where we can evolve each module separately and let users create their own modules.
By "consolidating" we are losing the modularity and ease of extensibility. We are overcrowding one contract with too much logic, locking the potential updates under the size limit.

I am open to hear the opposite argument though, if you have a good idea of how this can be improved and why.
also, which complications exactly you are talking about?

Reply:
I think they are different enough in the state structure they use vs the process with which they work. we do have some abstracted, and more work will be done to make it better. but I wanted to add full functionality first in order to make the best decision on what to abstract.